### ISSUE TITLE
docker, docker compose and rclone support?

### ISSUE THREAD

**Issue by aniel300** [DisboxApp/web#102]
any plans to support docker, docker compose and rclone support?

**Comment by 0bscenity** [comment]
im busy for the next few weeks, but after that, if i remember, i can try to follow [this guide](https://saqibullah.medium.com/convert-node-js-app-into-docker-container-4e3e0da70e17) to get docker up and running

**Comment by aniel300** [comment]
> im busy for the next few weeks, but after that, if i remember, i can try to follow [this guide](https://saqibullah.medium.com/convert-node-js-app-into-docker-container-4e3e0da70e17) to get docker up and running

yeah that should work and/or if this project can be made into an api that can then be added to rclone as a remote option then that would be even better. idk programming at the moment so i can't help u much

**Comment by aniel300** [comment]
@0bscenity any update on this? if this project could be made into a docker image to then deploy with docker compose and a way to access the files via webdav or ftp then adding to rclone should be easy after that. any thoughts?
